rules/linux/discovery_kernel_module_enumeration.toml

```diff
@@ -66,30 +66,33 @@ event.category:process and host.os.type:linux and event.type:start and event.act
  (process.name:(lsmod or modinfo)) or 
  (process.name:kmod and process.args:list) or 
  (process.name:depmod and process.args:(--all or -a))
-) and not process.parent.name:(mkinitramfs or cryptroot or framebuffer or dracut or jem or thin-provisioning-tools
-or readykernel or lvm2 or vz-start or iscsi or mdadm or ovalprobes or bcache or plymouth or dkms or overlayroot or 
-weak-modules or zfs)
+) and
+not (
+  process.parent.name:(
```

Comment:
can we do process.parent.name like () here since these are all specific strings?

Reply:
It's KQL, so unfortunately no